<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transferencias</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionRegisTrans"/>
    <addaction name="actionReporTrans"/>
    <addaction name="actionHistorialTrans"/>
   </widget>
   <widget class="QMenu" name="menuCreditos">
    <property name="title">
     <string>Creditos</string>
    </property>
    <addaction name="actionDesarrollador"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuCreditos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDesarrollador">
   <property name="text">
    <string>Desarrollador</string>
   </property>
  </action>
  <action name="actionRegisTrans">
   <property name="text">
    <string>Registrar Transferencia</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionReporTrans">
   <property name="text">
    <string>Reportar Transferencia</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionHistorialTrans">
   <property name="text">
    <string>Historial Transferencias</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
